port/cpl_http.cpp

```diff
@@ -921,16 +943,19 @@ int CPLHTTPPopFetchCallback(void)
 
 // NOTE: when adding an option below, add it in asAssocEnvVarOptionName[]
 
+// clang-format off
```

Comment:
Did you try ReflowComments false in clang-format options?

Reply:
> Did you try ReflowComments false in clang-format options?

I just tried ``ReflowComments: false``, but it doesn't seem to help

Reply:
Maybe also add: PenaltyBreakComment: 1000000000

Reply:
done in https://github.com/OSGeo/gdal/pull/6999 . That indeed fixed the weird reformatting of the Doxygen comments. But it had unpleasant consequences, where it generated invalid code (with '\' continuation character in comments), which I've manually fixed.